### PR TITLE
Add test case and function check about search/query output

### PR DESCRIPTION
**Pull request by NicoYuan1986**
#23075

### CODE REVIEW

File: tests/python_client/testcases/test_search.py [hunk 1]
```diff
@@ -2908,21 +2896,17 @@ def test_search_with_output_fields(self, nb, nq, dim, auto_id, _async):
...
-                                  check_items={"nq": nq,
-                                               "ids": insert_ids,
-                                               "limit": default_limit,
-                                               "_async": _async})[0]
-        if _async:
-            res.done()
-            res = res.result()
-        assert len(res[0][0].entity._row_data) != 0
-        assert (default_int64_field_name and default_float_field_name) in res[0][0].entity._row_data
+        output_fields = [default_int64_field_name, default_float_field_name]
```
Comment (binbinlv) [inline tests/python_client/testcases/test_search.py:2899]: Suggest to add value check for output_fields.

Comment (NicoYuan1986) [reply tests/python_client/testcases/test_search.py:2899]: I will add the value check in the next pr.

Comment (binbinlv) [reply tests/python_client/testcases/test_search.py:2899]: OK

**Pull request merged**